encoding/serialization.go

```diff
@@ -22,6 +23,41 @@ func (c *Frame) Deserialize(data []byte) (*Frame, error) {
 	return c, err
 }
 
+func (c *Frame) SerializeGnark() ([]byte, error) {
+	coded := make([]byte, 0, 32*(1+len(c.Coeffs)))
```

Comment:
can replace 32 with constants.go inside encoding/*

Reply:
for point size you can use SizeOfG1AffineUncompressed, see https://pkg.go.dev/github.com/consensys/gnark-crypto/ecc/bn254#G1Affine

Reply:
Done